module/metrics.go

```diff
@@ -191,12 +191,18 @@ type VerificationMetrics interface {
 	// It increases the total number of result approvals.
 	OnResultApproval()
 
-	// LogVerifiableChunkSize is called whenever a verifiable chunk is shaped for a specific
-	// chunk. It adds the size of the verifiable chunk to the histogram. A verifiable chunk is assumed
-	// to capture all the resources needed to verify a chunk.
-	// The purpose of this function is to track the overall chunk resources size on disk.
-	// Todo wire this up to do monitoring (3183)
-	LogVerifiableChunkSize(size float64)
+	// OnFinalizedBlockReceived is called whenever a finalized block arrives at the assigner engine.
+	// It increments the total number of finalized blocks.
+	OnFinalizedBlockReceived()
```

Comment:
The name was a bit confusing, since a node never receives a finalized block, but learned by itself that a block received earlier now got finalized.
```suggestion
	OnAssignerProcessFinalizedBlock()
```